Makefile

```diff
@@ -63,7 +63,7 @@ install_system_dependencies: apt_get_update ## Install system-level dependencies
 	sudo -E apt-get install -y `grep -v '^#' debian_packages.lst | tr -d '\r'`
 
 create_db: ## Create blanket DBs, i.e. `opencraft`.
-	createdb --encoding utf-8 --template template0 opencraft || \
+	createdb --host 127.0.0.1 --encoding utf-8 --template template0 opencraft || \
```

Comment:
This seems to be telling `createdb` to connect to the Postgres server via TCP rather than Unix domain sockets, right?  Will this still work when creating the devstack, i.e. is Postgres on Ubuntu configured to listen on localhost and allow connections without authentication?  I suspect not, but you can test it by trying to build a devstack from this branch.

Reply:
Tested it on vagrant and works.